### DISCUSSION TITLE
Pi Pico missing modules

### DISCUSSION THREAD

**Discussion by KevinStone01** [BIPES/BIPES#191] [Q&A] [answered]
I am trying to setup easyMqtt on the pico but I don't know how to get or install the missing module. here is what I get from the console.

ImportError: no module named 'umqtt'

Thanks in advance
 Kevin

**Comment by jetannenbaum** [comment, marked as answer]
Hi Kevin,

  Please try this version: https://bipes.net.br/pico/ui/  The Raspberry Pi Pico W has the proper library.  Since I only support the RP2040 based boards, I haven't tried to add the library to other boards.  If you are using another board, load the library with the Pico W selected, then switch to the board of your choice.  It should work.  Please let me know.

**Comment by jetannenbaum** [comment]
Sorry, should have read your issue a little more carefully.  The Pico version of the code will work for you.
